new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Docker.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.actions.impl.primitive;
+
```

Comment:
no imports?

Reply:
yes, no imports. The methods in the class executes certain commands, it uses other classes(Command, CommandParams) in the same package.